.github/workflows/main.yml

```diff
@@ -14,6 +14,21 @@ defaults:
     shell: bash -el {0}
 
 jobs:
+
+  shellcheck:
```

Comment:
Do we have any shell scripts to check?

Reply:
Only 1 for the moment (with some fixes from this same check).